frontend/src/hooks/useColumn.tsx

```diff
@@ -38,23 +38,35 @@ const useColumn = () => {
     onMutate: async (data) => {
       const prevBoard = await getPrevData(data.boardId);
 
-      // if(prevBoard)
-      // updateBoardColumns(data.boardId)
+      if (prevBoard) {
+        const columnsWithUpdate = prevBoard.columns.map((column) =>
+          column._id === data._id
+            ? {
+                _id: data._id,
+                color: data.color,
+                title: data.title,
+                cards: data.cards,
+                cardText: data.cardText,
+                isDefaultText: data.isDefaultText,
```

Comment:
```suggestion
            ...data
```
is it possible to do this way?